aplib.net-demo/Assets/Scripts/Camera/CameraManager.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraManager : MonoBehaviour
+{
+    [SerializeField] public Transform PlayerVisTransform;
```

Comment:
Because we specifically want a camera manager, this manager will probably also get more fields later on, right now when switching scenes or player objects etc, we can simply set the CameraManager field and nothing needs to be changed within the extension